src/utils/types.ts

```diff
@@ -96,11 +97,11 @@ export type secondaryLineConfigType = {
 export type referenceConfigType = {
   thickness?: number;
   width?: number;
-  color?: ColorValue | String | any;
-  type?: String;
+  color?: ColorValue | string | any;
+  type?: string;
   dashWidth?: number;
   dashGap?: number;
-  labelText?: String;
```

Comment:
It's advised to use `string`, not `String` in typescript because they represent different things.